src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java

```diff
@@ -128,8 +129,8 @@ protected ItemStack dispenseStack(IBlockSource source, ItemStack stack) {
                     }
                 }
             }
-            return new FishBucketDispenseBehavior().dispense(source, stack);
-        }
+            return stack;
+		}
```

Comment:
Everything else seems good.
This bracket still looks to be formatted incorrectly.

Reply:
Oh god it's still broken, hang on

Reply:
might not be broken now?